src/card/index.js

```diff
@@ -1,18 +1,17 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "./card.scss";
+import PropTypes from 'prop-types';
+import './card.scss';
 
 const Card = ({
-  imagePosition,
-  classNamePrefix,
-  cardOption,
-  renderImage,
-  data,
+  imagePosition = 'left',
+  classNamePrefix = null,
+  cardOption = null,
+  renderImage = true,
+  data = null,
```

Comment:
defaultProps is deprecated by React.

https://github.com/facebook/react/pull/16210